control/pid.go

```diff
@@ -291,10 +297,10 @@ func (p *pidTuner) computeGains() {
 		p.kP = (1.0 / (K * r)) * (4.0/3.0 + r/4)
 		p.kI = p.kP / (tauD) * (32 + 6*r) / (13 + 8*r)
 		p.kD = p.kP / (4 * tauD / (11 + 2*r))
-	default:
+	default: // ziegler nichols PI is the default
 		p.kP = 0.4545 * kU
 		p.kI = 0.5454 * (kU / pU)
-		p.kD = 0
+		p.kD = 0.
```

Comment:
why the dot?

Reply:
typing it explicitly as a float so that it matches all the other values set in this function. I can add a `0.0` like you've done elsewhere if you prefer that style.

Won't be a problem here, but I've noticed ogo treats not expressly typed floats as ints in some cases - we found a bug in gpio stepper where it was doing an integer division instead of a float division. I don't think this matters here because kD isn't used in the PI gain calculation, and x0 is 0, but I like this dot.